tokens/minne/color/focus-ring.yaml

```
@@ -0,0 +1,17 @@
+color:
+  focus-ring:
+    base:
+      light:
+        value: '{color.primitive.blue.500.value}'
+        attributes:
+          category: color
+          type: focus-ring
+          item: base
+          subitem: light
+      dark:
+        value: '{color.primitive.blue.400.value}'
```

`surface/base/dark`の`primary`と`secondary`に対してコントラスト比3.1以上を確保したいので、`200`への変更をお願いします🙏
```suggestion
        value: '{color.primitive.blue.200.value}'
```

---

いいですね、変えましょう！